pkg/repository/v1/scheduler_concurrency.go

```diff
@@ -68,7 +68,7 @@ func (c *ConcurrencyRepositoryImpl) UpdateConcurrencyStrategyIsActive(
 
 	defer rollback()
 
-	err = c.queries.ConcurrencyAdvisoryLock(ctx, tx, strategy.ID)
+	err = c.queries.AdvisoryLock(ctx, tx, strategy.ID)
```

Comment:
Renamed `ConcurrencyAdvisoryLock` to `AdvisoryLock` since it can be used wherever.